frontend/pages/DashboardPage/cards/MDM/MDMSolutionsTableConfig.tsx

```diff
@@ -58,16 +59,12 @@ export const generateSolutionsTableHeaders = (): IDataColumn[] => [
 ];
 
 export const generateSolutionsDataSet = (
-  solutions: IMdmSolution[] | null
-): IMdmSolution[] => {
-  if (!solutions) {
-    return [];
-  }
-
+  solutions: IMdmSolutionTableData[]
+): IMdmSolutionTableData[] => {
   return solutions.map((solution) => {
     return {
       ...solution,
-      displayName: solution.name ?? "Unknown",
+      displayName: solution.name || "Unknown",
```

Comment:
If I were starting from scratch, I'd probably try to move this transform into the `Cell` renderer in the table headers and remove the need for this function altogether. Maybe something like:

```tsx
  {
    title: "Name",
    Header: "Name",
    disableSortBy: true,
    accessor: "",
    Cell: ({ cell: { value }: ICellProps) => (
      <InternalLinkCell value={value || "Unknown"} />
    ),
  },
```

Sorting on the transformed value is the one place where it can get tricky. I don't know if that matters here since we are disabling sort on the name column. But  `generateTableData` approach is an easy enough workaround if it does matter in this case. The other option in that case would be to define the `sortType` for the column to take into account where how want the default value to fit into the sort. The latter might be something for us to consider adding to some stock column config object that we can override (similar to what we've been discussing for react-query).